src/mumble/JackAudio.h

```diff
@@ -159,8 +163,11 @@ class JackAudioOutput : public AudioOutput {
 	QMutex qmWait;
 	QSemaphore qsSleep;
 	JackPorts ports;
+	JackNamedPorts userPorts;
 	JackBuffers outputBuffers;
 	jack_ringbuffer_t *buffer;
+	void prepareOutputBuffers(unsigned int frameCount, QList< AudioOutputBuffer * > *qlMix,
```

Comment:
```suggestion

		void prepareOutputBuffers(unsigned int frameCount, QList< AudioOutputBuffer * > *qlMix,
```
(separation of variables and functions)

Reply:
Ah yeah, this looks nicer.